README.md

```diff
@@ -424,40 +465,53 @@ containing all arguments as specified above and the other attributes as specifie
 [`aws_s3_access_point`](https://www.terraform.io/docs/providers/aws/r/s3_access_point.html#attributes-reference) objects keyed by the `name` attribute.
 
 ## Module Versioning
+
 This Module follows the principles of [Semantic Versioning (SemVer)](https://semver.org/).
 
-Given a version number `MAJOR.MINOR.PATCH`, we increment the:
-1) `MAJOR` version when we make incompatible changes,
-2) `MINOR` version when we add functionality in a backwards compatible manner, and
-3) `PATCH` version when we make backwards compatible bug fixes.
+Using the given version number of `MAJOR.MINOR.PATCH`, we apply the following constructs:
+
+1. Use the `MAJOR` version for incompatible changes.
+1. Use the `MINOR` version when adding functionality in a backwards compatible manner.
+1. Use the `PATCH` version when introducing backwards compatible bug fixes.
+
+### Backwards compatibility in `0.0.z` and `0.y.z` version
 
-#### Backwards compatibility in `0.0.z` and `0.y.z` version
-- Backwards compatibility in versions `0.0.z` is **not guaranteed** when `z` is increased. (Initial development)
-- Backwards compatibility in versions `0.y.z` is **not guaranteed** when `y` is increased. (Pre-release)
+- In the context of initial development, backwards compatibility in versions `0.0.z` is **not guaranteed** when `z` is
+  increased. (Initial development)
+- In the context of pre-release, backwards compatibility in versions `0.y.z` is **not guaranteed** when `y` is
+  increased. (Pre-release)
```

Comment:
the old wording was actually way more readable as this is really important to make this clear to the point... now the context of the versions changed is harder to get as it is not visually comparable anymore... @soerenmartius we should definitely revert this back..

Reply:
opened a pr for this